app/models/concerns/memory_model.rb

```diff
@@ -1,14 +1,12 @@
-module NonPersistedModel
+module MemoryModel
```

Comment:
What about `ValueObject`?  These obects aren't doing anything, basically just wrappers around API responses.

Reply:
Well, there aren't used just for API responses.

The issue I have with using that is that for me Value objects should ideally be immutable objects which `ActiveModel::Model` aren't because it doesn't support read only, and in some use cases they behave like Rails like models with validations, etc

I used `MemoryModel` as a way to port one by one all the old Virtus classes, I could now rename it back to `NonPersistedModel`